### PR TITLE
Machinedata

### LINKED ISSUE #121: Wrong hardware data in baremetalhosts
This won't be included out-of-the-box until 4.3 but in the meantime we need to ship something.

Currently the hardware data is fake. It would be nice if the preflight script can inspect the hardware prior to installing it and when the baremetal <-> machine <-> node happens that data is used instead a fake one.

### PR DESCRIPTION
**Pull request by sreichar**
closes #121 Wrong hardware data in baremetal host

This will use correct data for the appliance, however false data still exist for when preflight did not gather the info using redfish

### CODE REVIEW

**Comment by sreichar** [comment]
[test]

**Comment by johnbieren** [comment]
[test]

File: OpenShift/99_post_install.sh [hunk 1]
```diff
@@ -79,7 +81,148 @@ create_ntp_config(){
...
+
+  if [ -z "${machine}" -o -z "${node_name}" ]; then
+      echo "Usage: $0 MACHINE NODE"
+      exit 1
+  fi
+
+  # BEGIN Hack #260
+  # Hack workaround for openshift-metalkube/dev-scripts#260 until it's done automatically
+  # Also see https://github.com/metalkube/cluster-api-provider-baremetal/issues/49
+  oc proxy &
```
Comment (johnbieren) [inline OpenShift/99_post_install.sh]: This PR hangs in CI. I think it has to do with this section of code. It seems something keeps the shell from exiting. The final output is
```
2019-09-16 05:38:47 | + oc wait mcp/worker --for condition=updated --timeout 600s
2019-09-16 05:38:47 | machineconfigpool.machineconfiguration.openshift.io/worker condition met
2019-09-16 05:38:47 | + kill_proxy
2019-09-16 05:38:47 | + kill 27154
2019-09-16 05:38:47 | ./99_post_install.sh: line 100: kill: (27154) - No such process
2019-09-16 05:38:47 | make[1]: Leaving directory '/home/cloud-user/install-scripts/OpenShift
```
and it sits there endlessly until the Jenkins job times out. Our task that calls make OpenShift never returns...
I have another PR running in CI to make sure it is specific to this PR

Comment (sreichar) [reply OpenShift/99_post_install.sh]: I am not sure I see anything wrong there,   there is a trap setup to make sure proxy is cleaned up on a exit or SIGINT,   I see the make leaves the directory.  So this has finshed.

**Comment by sreichar** [comment]
[test]   add kill_proxy at the end of each patch,  not sure, but a attempt

**Comment by sreichar** [comment]
[test]

**Comment by sreichar** [comment]
[test] lets see if trap reset works

**Pull request merged**